.travis/before_script_test.sh

```diff
@@ -1,3 +1,10 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
This one gets overwritten to `bash` by `dev-kit` on the next run.

Why did you change this?